programs/programs/nc/nc_lieap/calculator.py

```diff
@@ -0,0 +1,49 @@
+from programs.programs.calc import ProgramCalculator, Eligibility
+import programs.programs.messages as messages
+
+
+class NCLieap(ProgramCalculator):
+    expenses = ["rent", "mortgage", "heating"]
+    fpl_percent = 1.3
+    resource_limit = 2250
+    dependencies = [
+        "income_frequency",
+        "income_amount",
+        "zipcode",
+        "household_size",
+        "age",
+    ]
+    large_household_size = 3
```

Comment:
Based on the code `if household_size <= self.large_household_size:`, I think that the large household size is 4. You can then change your code to be `if household_size < self.large_household_size:`. I think that this is more accurate to the name of the variable because a household size of 3 is not the large household size, it is the biggest small household size.
```suggestion
    large_household_size = 4
```